app/views/claims/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<section>
+  <%= content_tag(:h1, "Claim This #{contribution.type}", class: "title") %>
+  <div class="box">
+    <p class="mb-2">
+      Information you enter here will be delivered directly to the other party.
+      Remember to protect yourself and never provide sensitive personal info to
+      strangers.
+    </p>
+        
+    <%= simple_form_for :peer_to_peer_match, url: contribution_claims_path(contribution) do |f| %>
```

Comment:
I'm curious about the choice to prefix the fields here with `peer_to_peer_match` instead of `claim`? I'm fine with, was mostly curious (typically, the form is named the same as the controller).

Reply:
Changed this to claim and refactored the controller layer too :)